internal/dockerfile/docker.go

```diff
@@ -54,6 +55,11 @@ func RenderConfig(cfg core.Configuration) {
 	var runVersionCommands []string
 	for _, binary := range cfg.Binaries {
 		if binary.InstallTo != "" {
+			if binary.InstallTo != "bin/" {
+				logg.Error("dockerfile: ignoring binary %q with custom install path %q, only 'bin/' is supported at the moment", binary.Name, binary.InstallTo)
```

Comment:
I would like to have a comment here explaining what the actual problem is. I'm assuming it's because the Dockerfile does a copy of `/pkg/bin`, right?